cr-examples/onnx/src/test/java/oracle/code/onnx/RuntimeTest.java

```diff
@@ -14,14 +15,16 @@ public class RuntimeTest {
     @Test
     public void test() throws Exception {
         var ort = OnnxRuntime.getInstance();
-        try (var absOp = ort.createSession(build(
-                List.of(tensorInfo("x", FLOAT.id)),
-                List.of(node("Abs", List.of("x"), List.of("y"), Map.of())),
-                List.of("y")));
+        try (Arena sessionArena = Arena.ofConfined()) {
```

Comment:
The same naming confusion is here. Below are created two sessions.

Reply:
> The same naming confusion is here. Below are created two sessions.

Well, this test creates an onnx session. The session constructor now takes an arena (the arena for the session) -- so the test now also creates an arena for that session and calls it "sessionArena".  I doesn't seem too confusing to me -- IMHO what you are really objecting to is the need of creating two things -- rather than the name?